src/Water/density.jl

```diff
@@ -0,0 +1,45 @@
+# these constant for used to computed density of air-free water
+const WD_A0 = 999.83952
+const WD_A1 = 16.945176
+const WD_A2 = -7.9870401e-3
+const WD_A3 = -4.6170461e-5
+const WD_A4 =  1.0556302e-7
+const WD_A5 = -2.8054253e-10
+const WD_AC =  0.016897850
+const WD_B0 = 999.85308
+const WD_B1 =  6.32693e-2
+const WD_B2 = -8.523829e-3
+const WD_B3 =  6.943248e-5
+const WD_B4 = -3.821216e-7
+
+
+
+
+"""
+    ρ_water_liq(tem::FT)
+
+Density of air-free pure water `[kg m⁻³]`, given
+- `tem` Water temperature `[K]`
+
+The equation for computing water density (0 to 150 Celcius) is
+```math
+ρ = \\frac{ 999.83952 + 16.945176 \\cdot t - 7.9870401e-3 \\cdot t^2 - 4.6170461e-5 \\cdot t^3 + 1.0556302e-7 \\cdot t^4 - 2.8054253e-10 \\cdot t^5 }{ 1 + 1.6897850e-2 \\cdot t }
```

Comment:
wow! Is it really so important to get the water density to such accuracy in the Land model? For clouds we just use a constant. 

And if yes, should we adapt the same in the atmosphere? So that we have consistent (if complicated) water density?

Reply:
The ocean model will need this T dependence for sure, should indeed make all of this consistent. I guess it matters mostly for the ocean but why not use the same everywhere.

Reply:
Fine with me. I think we should use one everywhere if we can

Reply:
It is different for sea water and pure water

Reply:
makes sense...  But then lets make sure that what we use for salty water converges to the not-salty version if salt-> zero

Reply:
Keep in mind that the land model deals only with *volume* fractions, hence assume water density to be constant. If you'd want to consider non-constant density, all land equations would have to be formulated as mass conservation equations rather than volume conservation equations. So: water density = constant is what we want in the land model.